test/unit/high-security.test.js

```diff
@@ -41,238 +35,278 @@ function getPath(obj, path) {
   return obj[paths[0]]
 }
 
-describe('high security mode', function () {
-  describe('config to be sent during connect', function () {
+tap.Test.prototype.addAssert('check', 3, function (key, before, after) {
+  var fromFile = { high_security: true }
```

Comment:
lets use const or let rather than var

Reply:
Def, I filed this https://github.com/newrelic/eslint-config-newrelic/issues/3 so in the future it won't be a PR comment but just enforced

Reply:
Good point, I'll make the change. (Since it was just two lines I also put up a PR for this eslint change: https://github.com/newrelic/eslint-config-newrelic/pull/4 )